wormhole-connect/src/hooks/useComputeSourceTokens.ts

```diff
@@ -71,7 +71,8 @@ const useComputeSourceTokens = (props: Props): ReturnProps => {
       active = false;
     };
     // IMPORTANT: do not include token in dependency array
-  }, [route, sourceChain, destToken, dispatch]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
what is it complaining about here? I don't see a variable named `token` anywhere.